Jenkinsfile

```diff
@@ -117,8 +117,8 @@ pipeline {
                                              aggregatingResults: true,
                                              id: "analysis-centos7",
                                              tools: [ gcc4(), cppCheck() ],
-                                             filters: [excludeFile('.*\\/_build\\.external-Linux\\/.*'),
-                                                       excludeFile('_build\\.external-Linux\\/.*')]
+                                             filters: [excludeFile(".*\\/_build\\.external${arch}\\/.*"),
+                                                       excludeFile("_build\\.external${arch}\\/.*")]
```

Comment:
Unless the [RFE that I filed upstream](https://issues.jenkins-ci.org/browse/JENKINS-55574) has been completed (but I didn't see my ticket closed) substituting `${arch}` doesn't work here.

Reply:
I think this is the difference between ' and ", it's simply how strings work in groovy.